### PR TITLE
Heads of Staff now have basic Maint access

### PR DESCRIPTION
**Pull request by Iamgoofball**
> be doctor turned CMO
> happen to be gang head
> respond to help cry in maint, have to be let in by the HoS
> no one's there, HoS fucks off
> i get stuck in there and run into a security room that opened because I can't get out through ANY OF THE DOORS CONNECTING TO PUBLIC HALLWAYS
> WITH A CMO ID
> HoS comes back, stuns me, searches me, finds my shit, executes on the spot
# salt

:cl: Iamsaltball
rscadd: Heads of Staff now have basic Maint. access.
/:cl:

### CODE REVIEW

**Comment by MMMiracles** [comment]
memes3

**Comment by coiax** [comment]
H U G B O X

**Comment by Iamgoofball** [comment]
@MMMiracles IC IN PR

**Comment by MMMiracles** [comment]
no idea what you're talking about

**Comment by KorPhaeron** [comment]
What LORE reason does the CMO have to be walking around in all the wires and pipes?

**Comment by Iamgoofball** [comment]
@KorPhaeron responding to medical emergencies in maint

**Comment by coiax** [comment]
Why doesn't the CMO send an underling to do that? CMO should be busy petting Runtime and doing paperwork.

**Comment by Iamgoofball** [comment]
what paperwork? there's no medical paperwork, also CMOs have a belt defib for dealing with late stage crit cardiac arrest, which can keep a guy alive for a little longer

immediate first aid can save a guy from dying

**Comment by MMMiracles** [comment]
i mean honestly all this does is cut out the extra minute a head of staff would of spent in the hop line/bridge getting the hop/captain to add maint access to their ID because literally why wouldn't you get maint access.

**Comment by LatD** [comment]
You could say the same thing for every other crew member as well, not just heads

**Comment by GunHog** [comment]
About bloody time :+1:

**Comment by Anonmare** [comment]
What L O R E  reason does a HoS have for having maint access? Doesn't he have underlings to search maint?

**Comment by Niknakflak** [comment]
I :-1: this simply because its grudgecode

**Comment by GunHog** [comment]
Goof should not posted the fact that this is grudgecode. It is legitimately a good change, in my opinion. Yes, Goof has the unfortunate history of succumbing to anger and memes, which clouds his achievements and defecates on his reputation. 

I feel that this PR, despite its origins, is a good change.

**Comment by KazeEspada** [comment]
They can already give themselves maint access using the card swipers.

**Comment by MMMiracles** [comment]
@KazeEspada 
>implying head cooperation isn't a literal unicorn event, let alone synchronized cooperation

**Comment by Shapsy** [comment]
Why are we talking about underlings, medical doctors don't have maint either

File: code/modules/jobs/job_types/security.dm [hunk 1]
```diff
@@ -25,11 +25,11 @@ Head of Security
 	access = list(access_security, access_sec_doors, access_brig, access_armory, access_court, access_weapons,
 			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_all_personal_lockers,
 			            access_research, access_engine, access_mining, access_medical, access_construction, access_mailsorting,
-			            access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway)
+			            access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway, access_maint_tunnels)
 	minimal_access = list(access_security, access_sec_doors, access_brig, access_armory, access_court, access_weapons,
 			            access_forensics_lockers, access_morgue, access_maint_tunnels, access_all_personal_lockers,
 			            access_research, access_engine, access_mining, access_medical, access_construction, access_mailsorting,
-			            access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway)
+			            access_heads, access_hos, access_RC_announce, access_keycard_auth, access_gateway, access_maint_tunnels)
```
Comment (Fox-McCloud) [inline code/modules/jobs/job_types/security.dm]: HoS already has maintenance access, ya dork.

**Comment by Fox-McCloud** [comment]
I'd rather HoS+detective lose maintenance access than this, TBH, but that's me.

File: code/modules/jobs/job_types/security.dm (hunk 1 above)
Comment (Shapsy) [inline code/modules/jobs/job_types/security.dm]: Only if sec maint access is enabled

Comment (Fox-McCloud) [inline code/modules/jobs/job_types/security.dm]: Look 2 lines above this one.

**Comment by xxalpha** [comment]
Why not.

**Comment by ChangelingRain** [comment]
abusing [s] to hide prs from the players is easily grounds for pr closure

**Comment by Iamgoofball** [comment]
it was just a prank tho

**Pull request closed without merging**